lib/translator.cpp

```diff
@@ -18,14 +18,126 @@
 #include "CL/cl_half.h"
 
 #include "opt/build_module.h"
+#include "opt/ir_context.h"
 #include "spirv/unified1/OpenCL.std.h"
 
 using namespace spvtools;
 using namespace spvtools::opt;
 using spvtools::opt::analysis::Type;
 
+namespace {
+
+std::string rounding_mode(SpvFPRoundingMode mode) {
+  switch (mode) {
+  case SpvFPRoundingModeRTE:
+    return "rte";
+  case SpvFPRoundingModeRTZ:
+    return "rtz";
+  case SpvFPRoundingModeRTP:
+    return "rtp";
+  case SpvFPRoundingModeRTN:
+    return "rtn";
+  }
+  return "UNKNOWN ROUNDING MODE";
+}
+
+const spvtools::MessageConsumer m_spvtools_message_consumer =
```

Comment:
This is no longer a member so you could drop the `m_` prefix and rename it.

Reply:
Makes sense, thanks for merging anyway and making the changes.